### PR TITLE
Adding kube-proxy download and pulling image with crictl

### PR DESCRIPTION
**Pull request by knabben**
1. Installing kube-proxy binary
2. Adding crictl
3. pre-pulling nanoserver image

### CODE REVIEW

File: experiments/image-builder/overlays/ansible/roles/utilities/tasks/main.yml [hunk 1]
```diff
@@ -13,8 +13,34 @@
...
+  win_get_url:
+    url: "https://github.com/kubernetes-sigs/cri-tools/releases/download/v1.22.0/crictl-v1.22.0-windows-amd64.tar.gz"
+    dest: '{{ programfiles.stdout | trim }}\containerd\crictl-v1.22.0-windows-amd64.tar.gz'
+  retries: 5
+  delay: 3
+
+- name: Download crictl and pull images
+  win_shell: |
+    tar xvf {{ programfiles.stdout | trim }}\containerd\crictl-v1.22.0-windows-amd64.tar.gz
+    {{ programfiles.stdout | trim }}\containerd\crictl --runtime-endpoint npipe:////./pipe/containerd-containerd pull mcr.microsoft.com/windows/nanoserver:1809
```
Comment (jsturtevant) [inline experiments/image-builder/overlays/ansible/roles/utilities/tasks/main.yml:42]: there is the ability to download additional images in image builder: https://github.com/kubernetes-sigs/image-builder/blob/master/images/capi/ansible/windows/roles/load_additional_components/tasks/registry.yml

Comment (knabben) [reply experiments/image-builder/overlays/ansible/roles/utilities/tasks/main.yml:42]: nice will use it.

**Pull request merged**